tests/nnfw_api/lib/GenModelTrain.h

```diff
@@ -66,7 +84,7 @@ static TrainCaseData uniformTCD(const std::vector<std::vector<T>> &inputs,
   for (const auto &data : inputs)
     ret.addInput(data);
   for (const auto &data : expects)
-    ret.addOutput(data);
+    ret.addExpects(data);
```

Comment:
I failed to catch why this PR has been requested. 

In my understanding, this PR replaces `TestCaseData.outputs` with `TrainCaseData.expects`. 
Is there a reason this change becomes necessary?

Reply:
The expected data is not the same as the output data. When this PR was introduced, the output buffers were not used. So the contributer of this change used the output buffer for the expected data. However, to calculate the accuracy in the training mode, the output buffer is required. Therefore, I introduce expects buffer for expected data.